### PR TITLE
Add missing word "database" in docs

### PR DESCRIPTION
**Pull request by tvsaru**
SQL is the language and based on second bullet point the author probably intended SQL "database".

---
Original PR #12274 , issuing this new PR via `bloomberg` org to leverage the company level CLA between Hashicorp and Bloomberg and also to comply with our internal policies on how to contribute to open source repos.

### CODE REVIEW

**Comment by hsimon-hashicorp** [comment]
@tvsaru for a corporate CLA, make sure that your GitHub handle matches what's in the CLA, and that your organization affiliation is publicly viewable. Thanks! :)

**Comment by tvsaru** [comment]
@hsimon-hashicorp I opened [my profile]() in incognito mode and I see the following:

> 
image

^ it shows I am part of @bloomberg org. Does that meet the requirements you noted?

**Comment by hsimon-hashicorp** [comment]
That satisfies one part of the bot, yes - but if your GitHub handle is different than what was provided in the corporate CLA, or you were added to the org after the CLA was signed, the bot might not be able to pick up on that. Can you check with your CLA signers to make sure you're included in the list? Thanks! :)

**Comment by tvsaru** [comment]
Oh I see. What I was looking at before was the employer field. I followed [GitHub doc](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership) to make my @bloomberg membership public. Lets see if that is enough for @hashicorp-cla bot.

image

**Comment by tvsaru** [comment]
@hsimon-hashicorp I asked the bot to recheck and LGTM now:

> 
image

**Comment by hsimon-hashicorp** [comment]
Huzzah! Teamwork for the win. Thanks @tvsaru for your patience there. :)

**Review by taoism4504 — APPROVED** [review]
LGTM

**Comment by tvsaru** [comment]
Thank @hsimon-hashicorp & @taoism4504. When will this get merged?

**Comment by hsimon-hashicorp** [comment]
@tvsaru I've kicked the tests that are failing, and will keep an eye on it. "Sometime today" is when I expect this to be merged. :)

File: website/content/docs/what-is-vault.mdx [hunk 1]
```diff
@@ -52,8 +52,8 @@ The key features of Vault are:
 
 - **Data Encryption**: Vault can encrypt and decrypt data without storing
   it. This allows security teams to define encryption parameters and
-  developers to store encrypted data in a location such as SQL without
-  having to design their own encryption methods.
+  developers to store encrypted data in a location such as an SQL database
```
Comment (taoism4504) [inline website/content/docs/what-is-vault.mdx:55]:
```suggestion
  developers to store encrypted data in a location such as a SQL database 
```

Comment (tvsaru) [reply website/content/docs/what-is-vault.mdx]: Thanks @taoism4504 . I did wonder why you originally changed 'a' to 'an' from my original commit. FYI, the discussion around `an` vs `a` before `SQL` and a list of other places in the docs where such inconsistencies exist can be found in [this comment](https://github.com/hashicorp/vault/pull/12274#issuecomment-942331973). Please consider fixing the others inconsistencies too.

Comment (taoism4504) [reply website/content/docs/what-is-vault.mdx]: Yes, an oversight on my part, and will fix the inconsistencies. Thank you.

**Comment by hsimon-hashicorp** [comment]
Hi @tvsaru - can you merge in main on this PR? It'll fix the automated test failures. I'm trying to work out an update to our process to not require some of these tests for docs-only changes, as well. Thanks in advance!

**Comment by tvsaru** [comment]
Sorry for the inconvenience @hsimon-hashicorp I should have rebased before issuing the new PR. Merged upstream/main into my branch now, 🤞🏽

**Comment by tvsaru** [comment]
@hsimon-hashicorp there is still one test failing:

> 
image

Maybe a [flaky test](https://app.circleci.com/pipelines/github/hashicorp/vault/24649/workflows/7df6f83a-521b-4f16-944f-162a071e79d9/jobs/363867/tests)?
> 
image

**Comment by hsimon-hashicorp** [comment]
That's one of our flakier tests, and doesn't block the merge. Thanks again for this PR! :)

**Pull request merged**